### ISSUE TITLE
Scroll wheel stuck on zoom after back gesture

### ISSUE THREAD

**Issue by agamotto** [Robbendebiene/Gesturefy#499]
### Describe your matter in detail
This seems to happen only on Plex server internal page - whenever I use default back gesture on this page, the scroll wheel starts zooming the page instead of actually scrolling. After switching to another tab and back the scroll wheel works again.

### Your System
- Operating system: Windows 10
- Firefox version: 77.0.1
- Gesturefy version: 3.0.2

**Comment by Robbendebiene** [comment]
If you can reproduce this bug on any other webpage please let me know. At the moment I'm not able to reproduce it.

> wheel starts zooming the page instead of actually scrolling

Do you mean the real Firefox page zoom, or some special zoom like behaviour?

**Comment by agamotto** [comment]
> 
> 
> If you can reproduce this bug on any other webpage please let me know. At the moment I'm not able to reproduce it.

Seems to happen on Newsblur as well.

> > wheel starts zooming the page instead of actually scrolling
> 
> Do you mean the real Firefox page zoom, or some special zoom like behaviour?

After reading this I tried disabling https://addons.mozilla.org/en-US/firefox/addon/zoom-page-we/ and the issue is not present anymore.

Now when I re-enable Zoom Page WE the scroll wheel is not zooming but does not scroll as well. If I right-click on the page to bring up the context menu then the scroll wheel is working again.

There seems to be a conflict with Zoom Page WE option -> "Enable mouse Right+Wheel to zoom in/out". After disabling it the issue is gone.

**Comment by Robbendebiene** [comment]
I've installed the extension you've mentioned, but I was not able to find an option called "Enable mouse Right+Wheel to zoom in/out".
Do you have any ideas why?

**Comment by shazzap** [comment]
I get the same problem. Actually, it happens on every page. 

What usually happens is:

I execute the gesture: I hold the right-mouse button and move the mouse up a bit to trigger the "go to top of page" gesture. I then take my hand completely off the mouse. 

Now I scroll the mouse wheel to scroll down the webpage. However it seems to trigger Zoom Page WE's  zoom-out function, making the page very small. 

The "Enable mouse Right+Wheel to zoom in/out" option is on that addon's General Tab, in the bottom section titled "User Interface.

**Comment by ElliotS-1** [comment]
Yeah I've got almost the same issue. It seems any gesture that doesn't change which page I'm on (back/forward don't affect it, not does new tab/close tab, but both scroll to top/bottom and zoom in/out do) will cause WE Zoom to start zooming with that option enabled. I've disabled the right click & scroll to zoom in it for now, and the problem is 'fixed'. 

Option is where Shazzap described, image attached to make it easier:
![option](https://user-images.githubusercontent.com/24664752/84794771-471cbe80-afbc-11ea-9277-7519185d2ef0.PNG)
